src/js/Controllers/UIController.js

```diff
@@ -203,6 +203,18 @@ class UIController {
                 }
                 this.listener.send(RpcFactory.SuccessResponse(rpc));
                 return null
+            case "UnsubscribeButton":
+                    store.dispatch(subscribeButton(
+                        rpc.params.appID,
+                        rpc.params.buttonName,
+                        false
+                    ));
+                    if(!rpc.params.buttonName || !rpc.params.appID) {
+                        this.listener.send(RpcFactory.ErrorResponse(rpc, GENERIC_ERROR, "No button provide to unsubscribe"));
```

Comment:
@LiliiaShlikhtLuxoft Please dont use declared `const` from other scope. Declare it globally

Reply:
DONE